src/openrct2-ui/windows/Window.h

```diff
@@ -101,8 +101,8 @@ void WindowGuestListRefreshList();
 WindowBase* WindowGuestListOpen();
 WindowBase* WindowGuestListOpenWithFilter(GuestListFilterType type, int32_t index);
 WindowBase* WindowStaffFirePromptOpen(Peep* peep);
-WindowBase* WindowScenarioselectOpen(scenarioselect_callback callback, bool titleEditor);
-WindowBase* WindowScenarioselectOpen(std::function<void(std::string_view)> callback, bool titleEditor);
+WindowBase* WindowScenarioselectOpen(scenarioselect_callback callback);
```

Comment:
Should remove this in a follow up PR, just need to change the callback signature to use std::string

Reply:
I'd rather leave that for a follow on